examples/next/pages/ui/components/liveness/components/useLiveness.ts

```diff
@@ -39,13 +48,19 @@ export function useLiveness() {
   const stopLiveness = () => {
     setLivenessActive(false);
     setGetLivenessResponse(null);
-    // mutate();
+    mutate();
   };
 
   const handleGetLivenessDetection = async (sessionId) => {
-    // const response = await API.get('BYOB', `/liveness/${sessionId}`, {});
-    // setGetLivenessResponse(response);
-    // return { isLive: response.isLive };
+    const response = await get({
+      apiName: 'BYOB',
+      path: `/liveness/${sessionId}`,
+      options: {},
+    }).response;
+    const { body } = response;
+    const livenessResponse = await body.json();
+    setGetLivenessResponse(livenessResponse);
+    return { isLive: (livenessResponse as any).isLive };
```

Comment:
Is it possible to have a type here?

Reply:
I'm not sure how to add generic typing to the new API calls but it seems to just return a type of `DocumentType`. This is also just in the sample app

Reply:
Fixed